readme.txt

```diff
@@ -223,6 +223,17 @@ The 2% fee on Stripe donations only applies to donations taken via our free Stri
 
 == Changelog ==
 
+= 2.6.3: May 19th, 2020 =
+* Fix: Improved Stripe's error handling so that the error output is more descriptive of the actual problem and donations are not marked as complete if the card is denied by Stripe. [#4689](https://github.com/impress-org/givewp/issues/4689)
+* Fix: When viewing legacy reports and filtering it would incorrectly redirect you to the new reports screen. [#4644](https://github.com/impress-org/givewp/issues/4644)
+* Fix: Alignment of the money fields were off a few pixels on the donation form settings screen. [#4638](https://github.com/impress-org/givewp/issues/4638)
+* Fix: Modified pro license logic so that old license keys saved to the database don't cause incorrect notices to display. [#4669](https://github.com/impress-org/givewp/issues/4669)
+* Fix: Modified pro license logic so that old license keys saved to the database don't cause incorrect notices to display. [#4669](https://github.com/impress-org/givewp/issues/4669)
+* Fix: Updated the country select dropdown to remove missing and incorrect countries. [#4745](https://github.com/impress-org/givewp/issues/4745)
+* Fix: Dates for the donot wall now print in the localized format. [#4721](https://github.com/impress-org/givewp/issues/4721)
```

Comment:
@DevinWalker We are not localizing date format instead of that we are printing the translated date. I think there are a difference between localizing format and translated.
The issue title also seems confusing to me.

Let me know what do you think.

Reply:
I updated this just now